src/components/TextComponent/TextComponent.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {
+    FC,
+    useState
+  } from 'react';
+// import "./TextInput.css";
+
+type TextInputProps = {
+    inputValue?:string,
+    placeholderText?: string,
+    error?:boolean,
+    onChange?:Function
+    // onChange?:React.ChangeEvent<HTMLInputElement>,
+    // All other props (...rest). Is this right???
+    // [x:string]: any;
+}
+
+const TextComponent: FC<TextInputProps> = ({
+    inputValue = "",
+    placeholderText = "",
```

Comment:
I think `placeholder` can be omitted, if you need to pass placeholder, it'll be `placehonder: "some text"` which will be handled by `...rest`